gremlin-go/NOTICE

```diff
@@ -0,0 +1,23 @@
+====================================================================================================
```

Comment:
Started this thread regarding LICENSE/NOTICE: https://lists.apache.org/thread/kgd52bz7pjc1qc638ktk31ooww5gbp8d